gateway/ln-gateway/src/lib.rs

```diff
@@ -1082,10 +1089,8 @@ impl Gateway {
                 prev_config.num_route_hints = num_route_hints;
             }
 
-            // TODO: Today, the gateway only supports a single routing fee configuration.
-            // We will eventually support per-federation routing fees. This configuration
-            // will be deprecated when per-federation routing fees are
-            // supported.
+            // Using this routing fee config as a default for all federation that has none
+            // routing fees specified.
```

Comment:
Clever, good backwards compatible way to make this work. Maybe we could change the name of `routing_fees` to `default_routing_fees` so its not confusing in the code which value is which